packages/icons/src/svg/shine_filled.svg

```diff
@@ -1,3 +1,3 @@
-<svg width="12" height="12" viewBox="0 0 12 12" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M9.81818 0L9.13091 1.5L7.63636 2.18182L9.13091 2.86909L9.81818 4.36364L10.5 2.86909L12 2.18182L10.5 1.5M4.36364 1.63636C3.8311 2.80794 3 4.63636 3 4.63636L0 6L3 7.36364L4.36364 10.3636L5.72727 7.36364L8.72727 6L5.72727 4.63636M9.81818 7.63636L9.13091 9.13091L7.63636 9.81818L9.13091 10.5L9.81818 12L10.5 10.5L12 9.81818L10.5 9.13091" fill="#231B22"/>
+<svg width="12" height="12" viewBox="0 0 12 12" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M9.81818 0L9.13091 1.5L7.63636 2.18182L9.13091 2.86909L9.81818 4.36364L10.5 2.86909L12 2.18182L10.5 1.5M4.36364 1.63636C3.8311 2.80794 3 4.63636 3 4.63636L0 6L3 7.36364L4.36364 10.3636L5.72727 7.36364L8.72727 6L5.72727 4.63636M9.81818 7.63636L9.13091 9.13091L7.63636 9.81818L9.13091 10.5L9.81818 12L10.5 10.5L12 9.81818L10.5 9.13091" />
 </svg>
```

Comment:
Do you want to use this optimization with [svgomg](https://jakearchibald.github.io/svgomg/)?:

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="12" height="12" viewBox="0 0 12 12">
  <path fill-rule="evenodd" d="m9.818 0-.687 1.5-1.495.682 1.495.687.687 1.495.682-1.495 1.5-.687-1.5-.682m-6.136.136L3 4.636 0 6l3 1.364 1.364 3 1.363-3 3-1.364-3-1.364m4.091 3-.687 1.495-1.495.687 1.495.682.687 1.5.682-1.5 1.5-.682-1.5-.687" clip-rule="evenodd"/>
</svg>
```
<img width="299" alt="Screenshot 2024-03-27 at 17 08 04" src="https://github.com/gympass/yoga/assets/796443/3191045f-4fa3-4f9a-9f1d-28228e123af0">

Reply:
The focus if this PR is only to fix a bug where the fill color wasn't being applied to this icon. It's not its focus to optimise the SVG itself.